cos-fleetshard-operator-debezium/src/test/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandControllerTest.java

```diff
@@ -67,13 +67,25 @@ public class DebeziumOperandControllerTest {
     private static final String SCHEMA_REGISTRY_ID = "9bsv0s0k8lng031se9q0";
     private static final String MANAGED_CONNECTOR_UID = "51eba005-daft-punk-afe1-b2178bcb523d";
     private static final String IMAGE_PULL_SECRET_NAME = "my-pullsecret";
+    private static final String APICURIO_AUTH_SERVICE_URL = "https://identity.api.openshift.com/auth";
```

Comment:
Maybe you want to test with some value that differs from the default values?

Reply:
@rk3rn3r I have used some different values in the recent commit, but it returns an NPE.